micro-rdk-server/Cargo.toml

```diff
@@ -29,6 +29,7 @@ micro-rdk = { workspace = true, features = ["esp32", "binstart", "provisioning"]
 
 
 [dependencies]
+micro-rdk-modular-driver-example = {path = "../examples/modular-drivers", features = ["esp32"]}
```

Comment:
using `micro-rdk-modular-driver-example.workspace = true` does not work, fails with an invalid workspace error.